server/controller/event.ts

```diff
@@ -25,6 +25,11 @@ export default class EventController {
     return evt
   }
 
+  public async editEvent(event: IEvent, organizationId: IEvent['organizationId']): Promise<void> {
+    this.deleteEvent(event.id, organizationId)
```

Comment:
This will delete the particular event completely before calling `this.create` Please double check

Reply:
editOrganization & editEvent works as intended now